gensrc/proto/tablet_schema.proto

```diff
@@ -91,6 +91,7 @@ message TabletSchemaPB {
 enum IndexType {
     BITMAP = 0;
     GIN = 1;
+    UNKNOWN_TYPE = 2;
```

Comment:
UNKNOW_TYPE is not a good name, because it is confusing. INDEX_UNKNOWN will be better.